src/examples/mfem_mortar_lm_patch.cpp

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include <set>
+
+#ifdef TRIBOL_USE_UMPIRE
+// Umpire includes
+#include "umpire/ResourceManager.hpp"
+#endif
+
+// MFEM includes
+#include "mfem.hpp"
+
+// Axom includes
+#include "axom/CLI11.hpp"
+#include "axom/slic.hpp"
+
+// Redecomp includes
+#include "redecomp/redecomp.hpp"
+
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
+#include "tribol/interface/tribol.hpp"
+
+int main( int argc, char** argv )
+{
+  // initialize MPI
+  MPI_Init( &argc, &argv );
+  int np, rank;
+  MPI_Comm_size(MPI_COMM_WORLD, &np);
+  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+
+#ifdef TRIBOL_USE_UMPIRE
+  umpire::ResourceManager::getInstance();  // initialize umpire's ResouceManager
+#endif
+
+  // initialize logger
+  axom::slic::SimpleLogger logger;
+  axom::slic::setIsRoot(rank == 0);
+
+  // command line options
+  // number of times to uniformly refine the serial mesh before constructing the
+  // parallel mesh
+  int ref_levels = 2;
+  // polynomial order of the finite element discretization
+  int order = 1;
+
+  axom::CLI::App app { "mfem_mortar_lm_patch" };
+  app.add_option("-r,--refine", ref_levels,
+    "Number of times to refine the mesh uniformly.")
+    ->capture_default_str();
+  app.add_option("-o,--order", order, 
+    "Finite element order (polynomial degree).")
+    ->capture_default_str();
+  CLI11_PARSE(app, argc, argv);
+
+  SLIC_INFO_ROOT("Running mfem_mortar_lm_patch with the following options:");
+  SLIC_INFO_ROOT(axom::fmt::format("refine: {0}", ref_levels));
+  SLIC_INFO_ROOT(axom::fmt::format("order:  {0}\n", order));
+
+  // fixed options
+  // location of mesh file. TRIBOL_REPO_DIR is defined in tribol/config.hpp
+  std::string mesh_file = TRIBOL_REPO_DIR "/data/two_hex_overlap.mesh";
+  // boundary element attributes of mortar surface
+  auto mortar_attribs = std::set<int>({4});
+  // boundary element attributes of nonmortar surface
+  auto nonmortar_attribs = std::set<int>({5});
+  // boundary element attributes of x-fixed surfaces
+  auto xfix_attribs = std::set<int>({1});
+  // boundary element attributes of y-fixed surfaces
+  auto yfix_attribs = std::set<int>({2});
+  // boundary element attributes of z-fixed surfaces
```

Comment:
Is there only one z-direction fixed surface? Maybe for clarity explain the {3,6}

Reply:
Surfaces at z = 0 and z = 1.95 planes.  Comment has been updated.